demos/rag_agentic/notebooks/Level6_agents_MCP_and_RAG.ipynb

```diff
@@ -560,20 +562,304 @@
     "6. Finally, the LLM sends the summarization as a message on Slack using the `slack_post_message` tool call configured with the Slack MCP server."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "39f6c84d",
+   "metadata": {},
+   "source": [
+    "## Defining our Agent - ReAct\n",
```

Comment:
To make it consistent with the changes in your level5 PR, can you also add a header before for the prompt chaining agent like:
`Defining our Agent - Prompt Chaining`